omas/tests/failed_imports.py

```diff
@@ -69,6 +69,8 @@
         failed_OMFIT = False
     except ImportError as _excp:
         failed_OMFIT = _excp
+    except AttributeError as _excp:
+        failed_OMFIT = "omfit_classes, from xarray import * bug "
```

Comment:
`omfit_classes` runs `from xarray import *` which is broken on various `xarray` versions.
I've reported & fixed this upstream, [see here](https://github.com/pydata/xarray/pull/8791). So with the next release of `xarray` this should be fixed again.